pkg/kube/controllers/boshdeployment/deployment_controller.go

```diff
@@ -85,7 +86,7 @@ func AddDeployment(ctx context.Context, config *config.Config, mgr manager.Manag
 	}
 
 	// Watch ConfigMaps referenced by the BOSHDeployment
-	configMapPredicates := predicate.Funcs{
```

Comment:
As for readability, why we need to shorten this variable and reuse it?

Reply:
Mostly because the other controllers also do that and for having shorter variable names: https://github.com/golang/go/wiki/CodeReviewComments#variable-names